core/src/main/scala/org/bitcoins/core/hd/BIP32Path.scala

```diff
@@ -144,6 +147,21 @@ object BIP32Path {
 
     BIP32PathImpl(path)
   }
+
+  def fromBytes(bytes: ByteVector): BIP32Path = {
```

Comment:
This seems like an important addition to BIP32Path, let's make sure we get some tests around this. This should be ripe for property based tests. We might be able to `extend Factory[BIP32Path]` on this companion object.

Reply:
We discussed this in a call, will be doing in a separate PR